src/chrome/content/rules/Aftenposten.xml

```diff
@@ -1,123 +1,27 @@
-		- kundeservice		(→ a)
-
-
-	Fully covered subdomains:
-
-		- a
-		- kundesenter
-		- kundeweb
-		- onlinesos
-
-
-	Insecure cookies are set for these domains:
-
-		- kundesenter.aftenposten.no
-		- www.aftenposten.no
-
-
-	Mixed content:
-
-		- css on www from $self ¹
-
-		- Images, on:
-
-			- leserpanel from $self ²
-			- www from $self ¹
-			- www from www.bt.no ¹
-			- www from ap.mnocdn.no ¹
-
-		- favicon on www from $self ¹
-
-		- Bugs on www from ocacache.medianorge.no ³
-
-	¹ Secured by us
-	² Not secured by us <= mismatched
-	³ Unsecurable
-
+	Mismatched:
+		- leserpanel
 -->
 <ruleset name="Aftenposten (partial)">
-
-        <target host="aftenposten.no" />
-        <target host="*.aftenposten.no" />
-		<!--
-			Redirects to http:
-						-->
-		<!--exclusion pattern="^http://www\.aftenposten\.no/$" /-->
-		<!--
-			Exceptions:
-					-->
-		<!--exclusion pattern="^http://www\.aftenposten\.no/(?!nyheter($|[?/]))" /-->
-		<!--
-			Avoid broken MCB:
-						-->
-		<exclusion pattern="^http://www\.aftenposten\.no/+(?!.+\.(?:css|gif|jpg|png)(?:$|\?)|\?service=css|favicon\.ico|skins/)" />
-	<target host="ap.mnocdn.no" />
-
-
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^kundesenter\.aftenposten\.no$" name="^_wp_session$" /-->
-	<!--securecookie host="^www\.aftenposten\.no$" name="^(VPW_Quota_\d+|VPW_QuotaInfo_\d+)$" /-->
-
-	<!--securecookie host="^\.aftenposten\.no$" name="^environment$" /-->
-	<securecookie host="^(?:a|kundesenter|kundeweb|onlinesos)\.aftenposten\.no$" name=".+" />
-
-
-	<rule from="^http://(?:www\.)?aftenposten\.no/"
-		to="https://www.aftenposten.no/" />
-
-	<rule from="^http://kundeservice\.aftenposten\.no/(?=$|\?.*)"
-		to="https://a.aftenposten.no/kundeservice/" />
-
-	<rule from="^http://(a|kundesenter|kundeweb|onlinesos)\.aftenposten\.no/"
-		to="https://$1.aftenposten.no/" />
-
-	<!--	The following rule will generate SSL warnings by default
-		because the cert is only valid for www.aftenposten.no.
-									-->
-	<!--rule from="^http://(lisacache|media|oslopuls)\.aftenposten\.no/"
-		to="https://$1.aftenposten.no/" /-->
-
-	<rule from="^http://ap\.mnocdn\.no/"
-		to="https://www.aftenposten.no/" />
-
+	<target host="aftenposten.no" />
+	<target host="www.aftenposten.no" />
+	<target host="bedrift.aftenposten.no" />
+	<target host="bil.aftenposten.no" />
+	<target host="kundesenter.aftenposten.no" />
+	<target host="kundeportal.aftenposten.no" />
+	<target host="kundeweb.aftenposten.no" />
+	<target host="lisacache.aftenposten.no" />
+	<target host="media.aftenposten.no" />
+	<target host="onlinesos.aftenposten.no" />
+	<target host="oslopuls.aftenposten.no" />
+	<target host="reise.aftenposten.no" />
+	<target host="rubrikkcache.aftenposten.no" />
+	<target host="sport.aftenposten.no" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http://sport\.aftenposten\.no/.*" to="https://www.aftenposten.no/100Sport/" />
+	<test url="http://sport.aftenposten.no/index.html" />
```

Comment:
I think this rule does not work as you expect. Please also add a comment for the mismatch on `sport.aftenposten.no`.

Reply:
@J0WI It's supposed to remove the path part of the URL.

Reply:
@J0WI Done.

Reply:
`.*` is not necessary, the regex will match anyway.

Reply:
@J0WI It would result in path not being removed, e.g. `https://www.aftenposten.no/100Sport/yhojszma7r6v2esfjk4ri5xo7q`